tests/Aspire.Hosting.Tests/Directory.Build.props

```diff
@@ -1,4 +1,7 @@
 <Project>
+  <PropertyGroup>
+    <IsNonHelixOnlyTestProject>true</IsNonHelixOnlyTestProject>
```

Comment:
The name feels weird. It would seem that `IsHelixOnlyTestProject=false` should be the same thing as this.

If we want to say "this test should never run on Helix", I would drop "Only" in the name. `IsNonHelixTestProject=true`.